cpp/src/parquet/arrow/reader.h

```diff
@@ -53,12 +53,15 @@ class ColumnReader;
 class RowGroupReader;
 
 static constexpr bool DEFAULT_USE_THREADS = false;
+static constexpr int64_t DEFAULT_BATCH_SIZE = 64 * 1024 * 1024;
```

Comment:
Is it a number of rows or a number of bytes?  Perhaps mention in a comment.

Reply:
Should be the number of rows, I'll add a comment.  Any thoughts on what a good default should be?

Reply:
I don't know. Perhaps 64k rows? 64M rows sounds huge. @wesm

Reply:
64K seems reasonable

Reply:
between 1k - 64k, see section 4.3 of https://pdfs.semanticscholar.org/ca7d/d279274916062a981c544da730e0e84abb87.pdf

Reply:
Thanks @fsaintjacques for the interesting paper, really useful to have some experimental data to refer to.  I'll press ahead with 64K as that was suggested in this thread and seems like fig 5 backs that up as a good choice.